Sources/NIOCore/Codec.swift

```diff
@@ -768,9 +768,16 @@ public final class MessageToByteHandler<Encoder: MessageToByteEncoder>: ChannelO
     private var state: State = .notInChannelYet
     private let encoder: Encoder
     private var buffer: ByteBuffer? = nil
+    private let maxBufferCapacity: Int?
 
+    public init(_ encoder: Encoder, maxBufferCapacity: Int) {
```

Comment:
This is no longer a `maxBufferCapacity` but rather the desired capacity to keep the buffer at.
```suggestion
    public init(_ encoder: Encoder, desiredBufferCapacity: Int) {
```

Reply:
Committed this change.